Content.Server/Administration/QuickDialogSystem.cs

```diff
@@ -139,7 +141,10 @@ private bool TryParseQuickDialog<T>(QuickDialogEntryType entryType, string input
                     return false;
                 }
 
-                output = (T?) (object?) input;
+                //It's verrrry likely that this will be longstring
+                var longString = (LongString) input;
+
+                output = (T?) (object?) longString;
```

Comment:
why do the other dialog types work then? this seems like the incorrect solution

Reply:
Give it a shot without this, it doesn't work otherwise.

Reply:
Was confused by this as well, but apparently this is required because its not a value type. E.g., see https://stackoverflow.com/questions/11738519/string-cast-not-working

Without the change it tries to do an actual cast which fails. With the change it will try to use the conversion operator.

All the casting shenanigans in this file are pretty cursed.